go/minterm/minterm.go

```diff
@@ -76,12 +73,38 @@ func (m *MinTerm) PromptPassword(prompt string) (string, error) {
 	if !strings.HasSuffix(prompt, ": ") {
 		m.Write(": ")
 	}
-	b, err := gopass.GetPasswd(int(m.termIn.Fd()))
+	return m.readSecret()
+}
+
+func (m *MinTerm) fdIn() int { return int(m.termIn.Fd()) }
+
+func (m *MinTerm) readLine() (string, error) {
+	fd := int(m.fdIn())
+	oldState, err := terminal.MakeRaw(fd)
+	if err != nil {
+		return "", err
+	}
+	defer terminal.Restore(fd, oldState)
+	var ret string
+	ret, err = terminal.NewTerminal(m.termIn, "").ReadLine()
+	return ret, convertErr(err)
+}
+
+func (m *MinTerm) readSecret() (string, error) {
+	fd := int(m.fdIn())
+	oldState, err := terminal.MakeRaw(fd)
 	if err != nil {
-		if err == gopass.ErrInterrupted {
-			err = ErrPromptInterrupted
-		}
 		return "", err
 	}
-	return string(b), nil
+	defer terminal.Restore(fd, oldState)
+	var ret string
+	ret, err = terminal.NewTerminal(m.termIn, "").ReadPassword("")
```

Comment:
Looks like we're doing the right thing with respect to #1637. Thanks for remembering that, @patrickxb!